test/testother.cpp

```diff
@@ -11332,6 +11334,20 @@ class TestOther : public TestFixture {
               "};\n");
         ASSERT_EQUALS("", errout.str());
     }
+
+    void knownPointerToBool()
+    {
+        check("void g(bool);\n"
+              "void f() {\n"
+              "    int i = 5;\n"
+              "    int* p = &i;\n"
+              "    g(p);\n"
```

Comment:
Will there be a warning for the equivalent `g(p != nullptr);`?

Reply:
I added warnings for those cases.